extensions/amp-accordion/amp-accordion.md

```diff
@@ -222,100 +215,167 @@ use its corresponding `id` as the value.
+  })();
+</script>
 ```
 
-[/example]
-
-### expand
-
+**expand**
 The `expand` action expands the sections of the `amp-accordion`. If a section
 is already expanded, it stays expanded. When called with no arguments, it
 expands all sections of the accordion. To specify a section, add the `section` argument, and use its corresponding `id` as the value.
 
 ```html
-<button on="tap:myAccordion.expand">Expand All Sections</button>
-<button on="tap:myAccordion.expand(section='section1')">
-  Expand Section 1
-</button>
+<button id="button1">Expand All Sections</button>
+<button id="button2">Expand Section 1</button>
+<script>
+  (async () => {
+    const accordion = document.querySelector('#myAccordion');
+    await customElements.whenDefined('amp-accordion');
+    const api = await accordion.getApi();
+
+    // set up button actions
+    document.querySelector('#button1').onclick = () => api.expand();
+    document.querySelector('#button2').onclick = () => api.expand('section1');
+  })();
+</script>
 ```
 
-### collapse
-
+**collapse**
 The `collapse` action collapses the sections of the `amp-accordion`. If a
 section is already collapsed, it stays collapsed. When called with no arguments,
 it collapses all sections of the accordion. To specify a section, add the
 `section` argument, and use its corresponding `id` as the value.
 
 ```html
-<button on="tap:myAccordion.collapse">Collapse All Sections</button>
-<button on="tap:myAccordion.collapse(section='section1')">
-  Collapse Section 1
-</button>
+<button id="button1">Collapse All Sections</button>
+<button id="button2">Collapse Section 1</button>
+<script>
+  (async () => {
+    const accordion = document.querySelector('#myAccordion');
+    await customElements.whenDefined('amp-accordion');
+    const api = await accordion.getApi();
+
+    // set up button actions
+    document.querySelector('#button1').onclick = () => api.collapse();
+    document.querySelector('#button2').onclick = () => api.collapse('section1');
+  })();
+</script>
 ```
 
-### Actions in Bento Mode
+#### Layout and style
 
-In Bento mode, the above actions can be called via `getApi()`. For example:
+Each Bento component has a small CSS library you must include to guarantee proper loading without [content shifts](https://web.dev/cls/). Because of order-based specificity, you must manually ensure that stylesheets are included before any custom styles.
 
-```html
-<script>
-  var api = await document.querySelector('#myAccordion').getApi();
-  api.toggle();
-  api.toggle('section1');
-</script>
 ```
+<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/amp-accordion-1.0.css">
+```
+
+Fully valid AMP pages use the AMP layout system to infer sizing of elements to create a page structure before downloading any remote resources. However, Bento use imports components into less controlled environments and AMP's layout system is inaccessible.
 
-[/filter] <!-- formats="websites" -->
+## Version notes
```

Comment:
Should this go to the bottom of the file?

Reply:
Thanks, moved to the bottom of the file!